assets/components/ctaLink/ctaLink.jsx

```diff
@@ -20,18 +22,24 @@ type PropTypes = {
 // ----- Component ----- //
 
 export default function CtaLink(props: PropTypes) {
+  const accessibilityHintId = props.id ? `accessibility-hint-${props.id}` : UUID();
+  const ctaUniqueClassName = `component-cta-link ${props.ctaId}`;
   return (
-    <a
-      id={props.id}
-      className="component-cta-link"
-      href={props.url}
-      onClick={props.onClick}
-      onKeyPress={props.onClick ? clickSubstituteKeyPressHandler(props.onClick) : null}
-      tabIndex={props.tabIndex}
-    >
-      <span>{props.text}</span>
-      <Svg svgName="arrow-right-straight" />
-    </a>
+    <span>
```

Comment:
The wrapper here should be a block level element, as it contains a block level (`<div>`) element.